astro.config.mjs

```diff
@@ -12,6 +12,14 @@ import remarkIncludeCode from "/src/plugins/remark-code-import";
 
 // https://astro.build/config
 export default defineConfig({
+  image: {
+    service: {
+      entrypoint: "astro/assets/services/sharp",
+      config: {
+        limitInputPixels: false,
+      },
+    },
+  },
```

Comment:
Without this change, some gifs were too big to be preprocessed and causes errors in build:

https://github.com/withastro/astro/pull/9546

Reply:
Can you pull this into a separate PR perhaps?